expenses-manager-frontend/src/main/frontend/src/app/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,23 @@
+<mat-sidenav-container class="sidebar">
+    <mat-sidenav>Start</mat-sidenav>
+    <mat-sidenav-content class="content">
+        <img src="../../assets/images/logo.svg" class="logo">
```

Comment:
As sugera sa adaugam atributul alt pentru imagine. Atributul alt pentru o imagine HTML () este utilizat pentru a specifica un text alternativ care să fie afișat atunci când imaginea nu poate fi încărcată sau pentru a fi citit de cititoarele de ecran pentru accesibilitate.

Reply:
În loc de a folosi aceeași clasă pentru imagine și text, ar fi mai bine să le diferențiezi.
Putem sa avem ceva de genul logo-image, logo-title, iar in css:
```
.logo-title, .logo-image {
 margin: 5px;
}
```